src/Nox.Generator/Application/Queries/TranslationsByIdQueryGenerator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.CodeAnalysis;
+using Nox.Generator.Application.Commands;
+using Nox.Generator.Common;
+using Nox.Solution;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Nox.Generator.Application.Queries;
+
+internal class TranslationsByIdQueryGenerator : ApplicationEntityDependentGeneratorBase
+{
+    protected override void DoGenerate(SourceProductionContext context, NoxCodeGenConventions codeGeneratorState, IEnumerable<Entity> entities)
+    {
+        var templateName = @"Application.Queries.TranslationsByIdQuery";
+        foreach (var entity in entities.Where(x => x.IsLocalized))
+        {
+            if (entity.IsOwnedEntity)
```

Comment:
owned entities will also be localized. no filter is needed

Reply:
Desired changes has been applied